.github/workflows/k6-action-image-release.yml

```
@@ -50,5 +50,7 @@ jobs:
       release_latest: false
       image_name: k6-action-image
       tag_prefix: k6-action-image-
-      workdir: ./infrastructure/images/k6-action
+      workdir: .
+      file: ./infrastructure/images/k6-action/Dockerfile
```

actionlint (1.7.4)

input "file" is not defined in "./.github/workflows/reusable-image-scan-and-release-ghcr.yml" reusable workflow. defined inputs are "image_name", "platforms", "release_latest", "tag_prefix", "workdir"

(workflow-call)